tests/test_other.py

```diff
@@ -6577,7 +6577,7 @@ def args_to_filename(args):
 
     if expected_size is not None:
       # measure the wasm size without the name section
-      self.run_process([wasm_opt, 'a.out.wasm', '--strip-debug', '-o', 'a.out.nodebug.wasm'])
+      self.run_process([wasm_opt, 'a.out.wasm', '--strip-debug', '--all-features', '-o', 'a.out.nodebug.wasm'])
```

Comment:
Will it be a problem if enabling all features slightly changes the size of the module? I know that at one point enabling bulk memory would increase the size by emitting a data count section, for instance.

Reply:
What I'm trying to do is say.. "accept all features in the input"... hopefully all this command does is remove the debugging sections..  I guess it can modify other parts of the file too simply be serializing the deserializing.   Hopefully it should be relatively stable though.   

Also, I don't see any alternative since this code is generic and should work on all kinds of different build outputs.  What other option do we have?

Reply:
Yeah, sounds good. If it does ever cause problems, then it would have exposed a real issue that would be good to fix anyway. LGTM!